lib/experimental/html/class-wp-html-tag-processor.php

```diff
@@ -1117,6 +1148,25 @@ public function get_tag() {
 		return strtoupper( $tag_name );
 	}
 
+	/**
+	 * Indicates if the current tag token is a tag closer.
+	 *
+	 * Example:
+	 * <code>
+	 *     $p = new WP_HTML_Tag_Processor( '<div></div>' );
+	 *     $p->next_tag( [ 'tag_name' => 'div', 'tag_closers' => 'visit' ] );
+	 *     $p->is_tag_closer() === false;
+	 *
+	 *     $p->next_tag( [ 'tag_name' => 'div', 'tag_closers' => 'visit' ] );
+	 *     $p->iss_tag_closer() === true;
+	 * </code>
```

Comment:
Nice example! A minor typo:

```suggestion
	 * Example:
	 * <code>
	 *     $p = new WP_HTML_Tag_Processor( '<div></div>' );
	 *     $p->next_tag( [ 'tag_name' => 'div', 'tag_closers' => 'visit' ] );
	 *     $p->is_tag_closer() === false;
	 *
	 *     $p->next_tag( [ 'tag_name' => 'div', 'tag_closers' => 'visit' ] );
	 *     $p->is_tag_closer() === true;
	 * </code>
```